### PR TITLE
Edits to GHCi, Editor Integration, and Bottoms

### PR DESCRIPTION
**Pull request by R-Morgan**
Here's another PR with some editing to the noted sections. As a note, the Bottoms section updates are in part a product of my own poking about on Stack Overflow and in the GHC documentation and trying to synthesize what I learned. Hopefully, these additions are factual and add clarity for beginning/intermediate users.

### CODE REVIEW

File: tutorial.md [hunk 1]
```diff
@@ -759,8 +763,8 @@ Data.Traversable fmapDefault :: Traversable t => (a -> b) -> t a -> t b
 Prelude fmap :: Functor f => (a -> b) -> f a -> f b
 ```
 
-For reasons of sexiness it is desirable to set your GHC prompt to a ``λ`` or a
-``ΠΣ`` if you're into that lifestyle.
+For reasons of aesthetic pleasure, it is desirable to set your GHC prompt to a ``λ`` or a
```
Comment (sdiehl) [inline tutorial.md]: This was intended to be slightly humorous. https://twitter.com/peterseibel/status/686319996146827264

File: tutorial.md [hunk 2]
```diff
@@ -818,47 +824,90 @@ remarkably improve the efficiency and productivity.
...
 undefined :: a
+
+
+mean :: Num a => Vector a -> a
+mean nums = (total / count) where            -- Partially defined function
+              total = undefined
+              count = undefined
+
+addThreeNums :: Num a => a -> a -> a -> a
+addThreeNums n m j = undefined               -- No function body declared at all
```
Comment (sdiehl) [inline tutorial.md]: This is a nice addition.

File: tutorial.md (hunk 1 above)
Comment (R-Morgan) [inline tutorial.md]: Haha, it's made it into the vocabulary of others. Keep it :).

**Comment by sdiehl** [comment]
This all looks great, I may revert the subtle joke about the GHCi prompt though. ;-)

Thanks again for all the progress on this, it's amazing to see so much work done so quickly.

**Comment by R-Morgan** [comment]
Thanks! It's so much for for me to see my work in public on a project I think is so cool :).

**Pull request merged**